static/js/TextsPage.jsx

```diff
@@ -171,8 +171,8 @@ const Dedication = () => {
         dedicationData && (dedicationData.en || dedicationData.he) ?
         <div className="dedication">
           <span>
-              <span className="int-en">{dedicationData?.en}</span>
-              <span className="int-he">{dedicationData?.he}</span>
+              <span className="int-en" dangerouslySetInnerHTML={ {__html: dedicationData?.en} }></span>
```

Comment:
can we at least run a function to sanitize the dedication data before sticking it here?

Reply:
I think this is 100% the right approach. I'm trying to hit a Monday deadline that was tossed on us this AM, but maybe I can harness the new pathway that exists for using Markdown in topic page descriptions. Gonna give that a shot.

Thanks for keeping me honest.